aspnet/migration/rc1-to-rc2.rst

```diff
@@ -28,64 +28,213 @@ You must update the SDK version in ``global.json``, as this file is used to conf
+
+.. code-block:: json
+
+  {
+    "frameworks": {
+      "netcoreapp1.0": {
+        "dependencies": {
+          "Microsoft.NETCore.App": {
+            "version": "1.0.0",
+            "type": "platform"
+          }
+        }
+      }
     }
   }
 
 Namespace and package ID changes
----------------------------------- 
+--------------------------------
 
-ASP.NET 5 was renamed to ASP.NET Core 1.0. Also, MVC and Identity are now part of ASP.NET Core. ASP.NET MVC 6 is now ASP.NET Core MVC. ASP.NET Identity 3 is now ASP.NET Core Identity.
+ASP.NET 5 has been renamed to ASP.NET Core 1.0. Also, MVC and Identity are now part of ASP.NET Core. ASP.NET MVC 6 is now ASP.NET Core MVC. ASP.NET Identity 3 is now ASP.NET Core Identity.
 
-All Microsoft.AspNet.\* packages and namespaces are renamed to Microsoft.AspNetCore.\*. 
-The EntityFramework.\* packages and namespaces are changing to Microsoft.EntityFrameworkCore.\*.
-All ASP.NET Core package versions are now 1.0.0-\*.
-Microsoft.Data.Entity.* is now Microsoft.EntityFrameworkCore.*
+All ``Microsoft.AspNet.*`` packages and namespaces have been renamed to ``Microsoft.AspNetCore.*``.
 
-Creating your web application host
-----------------------------------
+All ``EntityFramework.*`` packages and namespaces have been renamed to ``Microsoft.EntityFrameworkCore.*``. ``Microsoft.Data.Entity.*`` is now ``Microsoft.EntityFrameworkCore.*``.
 
-Since ASP.NET Core apps are just console apps, you must define an entry point for your application in ``Program.Main()`` that sets up a web host. Then tell the host to start listening. Below is an example of the startup code for the built-in `Web Application` template in Visual Studio.
+The ``EntityFramework.Commands`` package is no longer available. The ``ef`` command is now available via a .NET Core CLI tool in the ``Microsoft.EntityFrameworkCore.Tools`` package.
 
-.. code-block:: c#
+In addition, the following packages have been renamed:
+
+==========================================    ===================================================
+ASP.NET 5 RC1                                 ASP.NET Core 1.0
+==========================================    ===================================================
+EntityFramework.MicrosoftSqlServer            Microsoft.EntityFrameworkCore.SqlServer
+Microsoft.AspNet.Diagnostics.Entity           Microsoft.AspNetCore.Dianostics.EntityFrameworkCore
+Microsoft.AspNet.Identity.EntityFramework     Microsoft.AspNetCore.Identity.EntityFrameworkCore
+Microsoft.AspNet.Tooling.Razor                Microsoft.AspNetCore.Razor.Tools
+==========================================    ===================================================
+
+All ASP.NET Core package versions are now ``1.0.0``. Tooling packages used in ASP.NET Core 1.0 applications should use version ``1.0.0-preview2-final``.
+
+Commands and tools
```

Comment:
@danroth27, we spent some time previously to order the sections in order of relevance. I think we need to do that again with the newer changes. For example, in terms of migrating from RC1 -> RTM, I'm not sure if information regarding some of tools packages the should be above the webhost changes.